crates/uv-dev/src/generate_json_schema.rs

```diff
@@ -3,12 +3,24 @@ use std::path::PathBuf;
 use anstream::println;
 use anyhow::{bail, Result};
 use pretty_assertions::StrComparison;
-use schemars::schema_for;
+use schemars::{schema_for, JsonSchema};
+use serde::Deserialize;
 
 use uv_workspace::Options;
 
 use crate::ROOT_DIR;
 
+#[derive(Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+#[allow(dead_code)]
+// The name is used in the schema
+struct Uv {
+    #[serde(flatten)]
+    options: Options,
+    #[serde(flatten)]
+    dep_spec: uv_requirements::pyproject::Uv,
+}
```

Comment:
Hmm -- why isn't this change part of `uv_workspace` directly?

Reply:
I was thinking that pure configuration files shouldn't have dependency information, so keeping them separate